Src/WitsmlExplorer.Api/Workers/AnalyzeGapWorker.cs

```diff
@@ -52,20 +52,20 @@ public override async Task<(WorkerResult, RefreshAction)> Execute(AnalyzeGapJob
             return (new WorkerResult(GetTargetWitsmlClientOrThrow().GetServerHostname(), false, message), null);
         }
 
-        var jobMnemonics = job.Mnemonics.Any() ? job.Mnemonics : witsmlLog.LogCurveInfo.Select(x => x.Mnemonic);
-        await using LogDataReader logDataReader = new(GetTargetWitsmlClientOrThrow(), witsmlLog, jobMnemonics.ToList(), Logger);
+        var jobMnemonics = job.Mnemonics.Any() ? job.Mnemonics.ToList() : witsmlLog.LogCurveInfo.Select(x => x.Mnemonic).ToList();
+        await using LogDataReader logDataReader = new(GetTargetWitsmlClientOrThrow(), witsmlLog, new List<string>(jobMnemonics), Logger);
```

Comment:
jobMnemonics is used in the output as values selected by the user for analysis, and new instance is created, because the LogDataReader automatically adds an index mnemonic there.

Reply:
Makes sense